### DISCUSSION TITLE
No files for lidar detection models in Apollo 8.0

### DISCUSSION THREAD

**Discussion by IvDmNe** [ApolloAuto/apollo#15037] [Q&A] [answered]
Hello! I tried to test models from the tutorial: https://github.com/ApolloAuto/apollo/blob/master/docs/06_Perception/how_to_verify_lidar_function_review.md

For all models there are no files to test it.

Here is example of how the error looks like:

```
terminate called after throwing an instance of 'phi::enforce::EnforceNotMet'
  what():  (NotFound) Cannot open file /apollo/modules/perception/production/data/perception/lidar/models/detection/CenterPoint_paddle/centerpoint.pdmodel, please confirm whether the file is normal.
```

Where can we get models files?

**Comment by daohu527** [comment, marked as answer]
The perception model is too big, so we separated the model from the code. The download address is as follows.

Also, considering the convenience of installation, we have also developed a model management tool call [amodel](https://pypi.org/project/amodel/). For example, you can install the model in the following way
```
amodel install https://xxx.zip
```

Model | Model type | Model download address
-- | -- | --
PETR_V1_paddle | bev_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/petrv1.zip
tl_detection_caffe | trafficlight_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/tl_detection_caffe.zip
horizontal_torch | trafficlight_recognition | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/horizontal_torch.zip
quadrate_torch | trafficlight_recognition | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/quadrate_torch.zip
vertical_torch | trafficlight_recognition | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/vertical_torch.zip
denseline_caffe | lane_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/denseline_caffe.zip
cnnseg16_caffe | lidar_3d_segmentation | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/cnnseg16_caffe.zip
cnnseg64_caffe | lidar_3d_segmentation | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/cnnseg64_caffe.zip
center_point_paddle | lidar_3d_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/center_point_paddle.zip
mask_pillars_torch | lidar_3d_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/mask_pillars_torch.zip
point_pillars_torch | lidar_3d_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/point_pillars_torch.zip
3d-r4-half_caffe | camera_2d_detection | https://apollo-pkg-beta.cdn.bcebos.com/perception_model/3d-r4-half_caffe.zip

**Comment by IvDmNe** [comment]
Thank you, @daohu527 !
